akka-remote/src/main/scala/akka/remote/RemoteActorRefProvider.scala

```diff
@@ -257,11 +256,22 @@ private[akka] class RemoteActorRefProvider(
 
     // this enables reception of remote requests
     transport.start()
+    _akkaSystem = OptionVal.Some(transport.defaultAddress.toString)
```

Comment:
I find the name `akkaSystem` confusing, maybe call it `addressString`, even though the mdc value has the name `sourceActorSystem`